package.json

```diff
@@ -5,12 +5,14 @@
   "main": "init.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "wp-scripts start src/init.js src/link/index.js",
-    "build": "wp-scripts build src/init.js src/link/index.js",
```

Comment:
`src/init.js` still necessary, otherwise it doesn't compile in `build` folder

Reply:
You're right!